Content.Server/Power/EntitySystems/SubstationSystem.cs

```diff
@@ -121,15 +121,13 @@ public override void Update(float deltaTime)
     private void ConsumeNitrogenBoosterGas(float deltaTime, float scalar, SubstationComponent subs, PowerNetworkBatteryComponent battery, GasMixture mixture)
     {
         var initialN2 = mixture.GetMoles(Gas.Nitrogen);
-        var initialPlasma = mixture.GetMoles(Gas.Plasma);
 
         var molesConsumed = (subs.InitialNitrogenBoosterMoles * battery.CurrentSupply * deltaTime) / (_substationDecayCoeficient * scalar);
 
-        var minimumReaction = Math.Min(initialN2, initialPlasma) * molesConsumed / 2;
```

Comment:
Looks like you deleted too much here. You still want _initialN2 * molesConsumed / 2;_

Reply:
Huh thought to remove the division since it's no longer two gasses but I guess that's the consumption bit. Thanks

Reply:
Corrected to:
```
        var minimumReaction = Math.Abs(initialN2) * molesConsumed / 2;

        mixture.AdjustMoles(Gas.Nitrogen, -minimumReaction);
        mixture.AdjustMoles(Gas.NitrousOxide, minimumReaction);
```
Was getting errors without Math. and with Math.min so settled on .Abs to get no errors. Doesn't seem to be using the gas still so going to give it some time.